src/LinearLoader/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+import { theme } from '../theme.js'
+
+const Progress = ({ amount }) => {
+    return (
+        <div>
+            <style jsx>{`
+                div {
+                    width: ${amount}%;
+                }
+            `}</style>
+            <style jsx>{`
+                div {
+                    background-color: ${theme.primary600};
+                    transition: width 0.3s linear;
+                    height: 4px;
+                }
+
+                .determinate {
+                }
+            `}</style>
+        </div>
+    )
+}
+
+const LinearLoader = ({ amount, width, margin, className }) => {
+    return (
+        <div role="progressbar" className={className}>
+            <Progress amount={amount} />
+
+            <style jsx>{`
+                div {
+                    display: block;
+                    overflow: hidden;
+                    overflow-x: hidden;
+                    overflow-y: hidden;
+                    background-color: rgba(110, 122, 138, 0.2);
+                }
+            `}</style>
+            <style jsx>{`
```

Comment:
Both `style` tabs cam be combined

Reply:
No, the second one is dynamic since it uses `props`.